Documents/Changelog/Changelog.md

```diff
@@ -3,7 +3,7 @@
 ==== 
 
 ## 2025-11-xx - Build 2511 (V10 - alpha) - November 2025
-* Implemented [#2231](https://github.com/Krypton-Suite/Standard-Toolkit/issues/2231), Use `filescoped` namespaces
+* Implemented [#2231](https://github.com/Krypton-Suite/Standard-Toolkit/issues/2231), **[Breaking Change]** Use `filescoped` namespaces
```

Comment:
Not a breaking change unless they have also changed the namespace (Which should not have happened !!)

Reply:
Will remove in a future PR